NEWS.md

```diff
@@ -1,5 +1,13 @@
 # NEWS for Lrama
 
+## Lrama 0.7.1 (2025-xx-xx)
+
+### Syntax Diagrams
```

Comment:
It might good to add how to specify the option.

Reply:
Indeed it is.  I updated: https://github.com/ruby/lrama/pull/523/commits/4280ca2aac11b7173a94ba95e03d827483ba2bd8